bundle/config/resources/dashboard.go

```diff
@@ -0,0 +1,59 @@
+package resources
+
+import (
+	"context"
+
+	"github.com/databricks/cli/libs/log"
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/marshal"
+	"github.com/databricks/databricks-sdk-go/service/dashboards"
+)
+
+type Dashboard struct {
```

Comment:
What should happen if you have a dashboard in a DAB but don't reference it as a resource?